model/src/main/java/uk/gov/pay/commons/model/ErrorIdentifier.java

```diff
@@ -5,5 +5,6 @@
     REFUND_NOT_AVAILABLE,
     REFUND_AMOUNT_AVAILABLE_MISMATCH,
     ZERO_AMOUNT_NOT_ALLOWED,
-    GO_CARDLESS_ACCOUNT_NOT_LINKED
+    GO_CARDLESS_ACCOUNT_NOT_LINKED,
+    GO_CARDLESS_ACCOUNT_ALREADY_USED
```

Comment:
`GO_CARDLESS_ACCOUNT_ALREADY_LINKED`? (I’d prefer `CONNECTED` because that’s what GoCardless call it but consistency and all that)

Reply:
How about `GO_CARDLESS_ACCOUNT_LINKED_TO_ANTHER_ACCOUNT`?

Reply:
I like having `ALREADY` in there so maybe `GO_CARDLESS_ACCOUNT_ALREADY_LINKED_TO_ANOTHER_ACCOUNT`?

Reply:
Ok changed to that